docs/index.md

```diff
@@ -75,13 +71,7 @@ Install Codacy on an existing cluster using our Helm chart:
 6.  <span id="helm-upgrade">Add Codacy's chart repository to your Helm client and install the Codacy chart using the file `values-production.yaml` created previously.</span>
 
     !!! important
-        **If you are using MicroK8s** you must use the file `values-microk8s.yaml` together with the file `values-production.yaml`.
-
-        Use `wget` to download the extra file and uncomment the last line before running the `helm upgrade` command below:
-
-        ```bash
-        wget https://raw.githubusercontent.com/codacy/chart/master/codacy/values-microk8s.yaml
```

Comment:
I'd remove this command because it forces you to write the path to a static file, or use more complicated parsing just for the sake of teaching someone how to download a file.